app/app.js

```diff
@@ -17,6 +17,7 @@ const AppFactory = function() {
 
     global.DOWNLOAD_TOTAL_SIZE = 0;
     global.PROCESSING_LIST = new Set();
+    global.UNAVAILABLE_SPACE_LIST = new Set();
```

Comment:
Why is it global? Isn't it better to make it a property of `fileSystem`?

Reply:
@andrecardoso It is a global set for the entire app as the file route creates a new instance of file controller per every request for a file. It's not a property of file system as that file was designed to just provide helper functions.